KaylaKornelis-app/src/main/java/edu/KaylaKornelis/advancedjava/Assignment3/StockQuoteApplication.java

```diff
@@ -0,0 +1,60 @@
+package edu.KaylaKornelis.advancedjava.Assignment3;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Calendar;
+import java.util.Date;
+import java.util.List;
+
+/**
+ * This class is used to retrieve stock quotes
+ */
+public class StockQuoteApplication {
+    
+    public static void main(String[] args){
```

Comment:
don't forget to document all methods including this main method.

Reply:
Added javadoc throughout.